docs/maintainer/knowledge_base.md

```diff
@@ -533,7 +533,6 @@ requirements:
     - numpy
   run:
     - python
```

Comment:
```suggestion
    - python
    - numpy >=YOUR_MINIMUM_RUNTIME_REQUIREMENT_FOR_NUMPY
```

Reply:
I'm against this; this is under the heading "A simple Python extension [...]", and in the default case, the runtime requirement should be handled entirely by numpy's run-export. The mechanics are explained in the respective section, I'm happy to add a reference to that.

Reply:
ok